### PR TITLE
resolves issue #143, adding CRI support

### PR DESCRIPTION
**Pull request by jeffywu**
Not sure if what I did here makes 100% sense, but I think the changes are pretty straightforward. Can move where the values are stored to a path that makes more sense.

### CODE REVIEW

**Comment by jeffywu** [comment]
@chaitanyaphalak, made some changes to the branch. Let me know if it looks better.

**Comment by jeffywu** [comment]
@chaitanyaphalak, good catch! updated that section.

**Comment by chaitanyaphalak** [comment]
@jeffywu can you rebase this PR to the latest develop branch. Thanks.

**Comment by chaitanyaphalak** [comment]
Needs a rebase with develop @jeffywu

File: helm-chart/splunk-kubernetes-logging/templates/daemonset.yaml [hunk 1]
```diff
@@ -74,10 +74,10 @@ spec:
       volumes:
       - name: varlog
         hostPath:
-          path: /var/log
-      - name: varlibdockercontainers
+          path: {{ .Values.containers.path }}
+      - name: varlogdest
         hostPath:
-          path: {{ with .Values.dockerPath }}{{ printf "%s/containers" . | quote }}{{ end }}
```
Comment (jeffywu) [inline helm-chart/splunk-kubernetes-logging/templates/daemonset.yaml]: @chaitanyaphalak, was a little confused what to do with this line here. Happy to use .Values.dockerPath but having it append "/containers" automatically won't work for me, I need this to be set to "/var/data/kubeletlogs"

Comment (chaitanyaphalak) [reply helm-chart/splunk-kubernetes-logging/templates/daemonset.yaml]: I am ok with your implementation. Its more generic.

**Pull request merged**